tests/functional/api/test_projects.py

```diff
@@ -244,7 +244,7 @@ def test_project_protected_branches(project):
 
 
 def test_project_remote_mirrors(project):
-    mirror_url = "http://gitlab.test/root/mirror.git"
+    mirror_url = "https://gitlab.com/root/mirror.git"
```

Comment:
> I think push mirrors don't create any requests by default, but just in case:

Yeah push mirroring is disabled by default:
https://docs.gitlab.com/ee/api/remote_mirrors.html#create-a-push-mirror

But a good idea to use a non-real host-name. Let me update it and test locally to make sure it works.

Reply:
We enable it a few lines down from here, not sure if has any effects still, but just in case. Thanks :)

Reply:
Ah. I wasn't looking at that. Nice that the test works even with a not existing host.